### DISCUSSION TITLE
Kinoite and Opensnitch via Boxbuddy (Distrobox)

### DISCUSSION THREAD

**Discussion by steelowheel** [evilsocket/opensnitch#1226] [Q&A] [answered]
Hello, can anyone confirm Opensnitch works on Kinoite? I just freshly installed Kinoite (actually Ublue Aurora) and then installed Opensnitch and its UI in BoxBuddy. But when I open the UI it doesn't show connections, apps or just about anything. Also I was expecting to start to get allow/deny prompts right away, but I haven't seen any. I'm new to Linux, perhaps I missed something? Any help will be appreciated. Thanks!

Edit: attached screenshots
![opensnitch1](https://github.com/user-attachments/assets/3a8c0007-81ff-42ff-a519-1f5d9fd8a779)
![opensnitch2](https://github.com/user-attachments/assets/4e92747c-2159-4030-8f6a-8e8bfdca7709)
![opensnitch3](https://github.com/user-attachments/assets/302698e8-684e-42de-a571-8bf7f631cf20)
![opensnitch4](https://github.com/user-attachments/assets/2581f606-010d-453c-8b07-5b0622ceafeb)
![Screenshot_20241219_180240](https://github.com/user-attachments/assets/bc22bed7-535d-456d-b39e-51e3028bc757)

**Comment by AverageHelper** [comment]
Same here, lol. I'll report back if I can get it to do something on Kinoite other than this empty UI.

(EDIT: i finished an edit just as you posted, so made it a reply instead :sweat_smile:)

**Reply by steelowheel** [reply]
> Same here, lol. I'll report back if I can get it to do something on Kinoite other than this empty UI lol

I noticed that if I tick the automatic startup in OpenSnitch UI settings, it doesn't start with the system or login.

I also tried rpm-ostree install, but with same results: empty events, nodes etc. But it seems that when installed via rpm-ostree, OpenSnitch UI starts with login.

**Reply by AverageHelper** [reply, marked as answer]
For my case, it looks like I missed that we need to install _both_ `opensnitch` and `opensnitch-ui` for the UI to do anything. So basically: look for 2 .rpm files in the [latest Release(s)](https://github.com/evilsocket/opensnitch/releases), named something like:
- `opensnitch-x.x.x-x.x86_64.rpm`
- `opensnitch-ui-x.x.x-x.noarch.rpm`

You can install these with `rpm-ostree install`.

Then, after a reboot, you enable the `opensnitch` service:
```sh
systemctl enable --now opensnitch
```
and hope and pray that systemd actually restarts the service on reboot, lol

(i rebooted just now and it looks like it did, so _yay_)

**Reply by steelowheel** [reply]
I installed both opensnitch and opensnitch-ui. But I didn't do this:

> Then, after a reboot, you enable the `opensnitch` service:
> 
> ```shell
> systemctl enable --now opensnitch
> ```

... because I don't even know what systemctl is :D But thank you, I'll try this and see how it goes.

**Edit:**
Okay, in my case it's a matter of rtfm. 
Quote from https://github.com/evilsocket/opensnitch/wiki/Installation : 

> Remember:
> 
> If the daemon doesn't start after installing these packages (or others from other distributions) or after rebooting your computer, you need to enable and start it manually:
> 
> sudo systemctl enable --now opensnitch
> sudo systemctl start opensnitch
> (use opensnitchd if opensnitch doesn't work)

**Reply by AverageHelper** [reply]
yeah, basically that `systemctl` command tells Systemd to run the `opensnitch` daemon on startup, and also to run it right away if you use the `--now` flag or that `start` subcommand. I guess the .rpm files can't do that magic dance automatically in the install, so it's up to users for now, lol
